client/.eslintrc.js

```diff
@@ -25,11 +25,22 @@ module.exports = {
     sourceType: "module", // Allows for the use of imports
   },
   rules: {
+    "no-prototype-builtins": "off",
     "react/display-name": "off",
+    "react/prop-types": "off", // Disable prop-types as we use TypeScript for type checking
```

Comment:
i want to get rid of all these TS-rule exceptions. Probably I got this stuff added by copying from elsewhere but we're moving in general towards stricter typing and linting. We can look/discuss tmrw